embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/api/EmbraceUrl.kt

```diff
@@ -11,6 +12,17 @@ internal class EmbraceUrl(val url: URL) {
         return EmbraceConnectionImpl(url.openConnection() as HttpURLConnection, this)
     }
 
+    fun toEndpoint(): Endpoint? {
```

Comment:
what about just calling it `endpoint()` since we're not transforming the object?

Reply:
yeap, that's better, updated.